data/heliarch intro.txt

```diff
@@ -0,0 +1,1568 @@
+					goto back
+			label bushending
+			`	You help him unearth the bushes, and dig a bit more to see if there's anything under them, but no matter how much you dig, you never find anything. You find some other suspicious sets of bushes, but none have anything underneath. The rain keeps getting heavier and the wind more powerful, so you two head to a warehouse and take shelter there.`
+			`	Once the weather settles, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back as well.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label back
+			`	Though he wanted to try digging them out, you convince Kirrie to leave the bushes alone, and you two head back to the empty streets, surrounded by the warehouses. You can barely differentiate the warehouses from earlier, as the rain gets even heavier.`
+			choice
+				`	(Check the warehouses with the rusty doors.)`
+					goto rustyending
+				`	(Check the warehouses with infiltration.)`
+					goto reactor
+			label rustyending
+			`	Kirrie uses his card to open the door, and continues to explain that it is an access card granted to Heliarch agents, in case they need to do something like this, or just identify themselves as one.`
+			`	Sadly the warehouse turns out to not contain anything significant. It's a storage for what Kirrie says are toy parts, just mountains of plastic stacked together.`
+			`	You check a few more like this, but none turn out to have anything, and as the rain and wind become too much to walk around, you take shelter in one of the warehouses.`
+			`	Once the weather settles, near nightfall, he gets notified by Iplora that she and Tugroob are already back at the ship, so you two decide to head back as well.`
+			`	It seems Tugroob found some evidence of some smuggled weapons passing through here, but nothing major. They get back in their bunks, saying you may depart now.`
+				accept
+			label trees2
+			`	You suggest looking for something unusual near the plant life you spotted, while the rain is still fairly tame, and he follows you. You spend a few minutes looking around the trees, but find nothing suspicious.`
+			`	While passing by another set of bushes, he notices that some of them are awkwardly tilted. Upon further inspection, he tells you that they appear to have been removed, and then forcibly put on the ground again, noting some odd, wavy patterns of the small rocks around the roots. The rain gets even heavier.`
+			choice
+				`	(Dig out the bushes.)`
+					goto bushending
+				`	(Head back to the open warehouses and take shelter there.)`
+					goto beetledigger
+			label beetledigger
+			`	Though you can barely hear each other in the rain, Kirrie insists in staying there and digging through the ground for a while, to see if he finds anything.`
```

Comment:
`	Though you can barely hear each other in the rain, Kirrie insists on staying there and digging through the ground for a while to see if he finds anything.`